diff --git a/node_modules/expo-gl-cpp/android/build.gradle b/node_modules/expo-gl-cpp/android/build.gradle
index 025d491..d5bde51 100644
--- a/node_modules/expo-gl-cpp/android/build.gradle
+++ b/node_modules/expo-gl-cpp/android/build.gradle
@@ -50,7 +50,7 @@ buildscript {
 
 // Creating sources with comments
 task androidSourcesJar(type: Jar) {
-  classifier = 'sources'
+  archiveClassifier.set('sources')
   from android.sourceSets.main.java.srcDirs
 }
 
@@ -72,6 +72,7 @@ afterEvaluate {
 }
 
 android {
+  compileSdk 34 
   compileSdkVersion safeExtGet("compileSdkVersion", 31)
 
   compileOptions {
